### PR TITLE
#3 Add travis

### PR DESCRIPTION
**Pull request by gkman**

### CODE REVIEW

**Comment by gkman** [comment]
@simsalabim I need help. I'm doing something wrong.

File: .rubocop.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ AllCops:
   UseCache: false
   Exclude:
    - 'projects/**/*'
+   - 'vendor/**/*'
```
Comment (simsalabim) [inline .rubocop.yml]: @gkman that's what you were missing. How do I know? I made `.travis.yml` like so at some point
```yaml
script:
  - pwd
  - ls -la
```
which revealed the `vendor` directory Travis CI creates in the workspace directory. Rubocop was failing on files in that directory. Hence the fix was to exclude `vendor`

Comment (gkman) [reply .rubocop.yml]: got it

**Comment by gkman** [comment]
👍

**Pull request merged**